src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx

```diff
@@ -25,6 +25,21 @@ curl -X GET 'https://api.newrelic.com/v2/applications.json' \
 -d "filter[name]=<var>${APP_NAME}</var>"
 ```
 
+## Get a Single application ID from a List filtered by application name [#get_app_id_from_name]
+
+You need to get an ID for an app that you have by name, or potentially do not have at all with a little you can pull just the id you need .
```

Comment:
Could you please review this sentence? I don't understand it well. Is the word 'little' what you wanted to write?

Reply:
The part that should be there is 'jq'. I am fixing it right now. I've been offline most of the holiday.

Reply:
OK, I'll merge it after this change. Thanks!

Reply:
Hi @shanechambers-ds! Could you please add the change to merge it? Thanks!